pilbox/app.py

```diff
@@ -138,7 +138,7 @@ def __init__(self, **kwargs):
         tornado.web.Application.__init__(self, self.get_handlers(), **settings)
 
     def get_handlers(self):
-        return [(r"/", ImageHandler)]
+        return [(r"/(.*)", ImageHandler)]
```

Comment:
Is this the only functional change? If so, can we not bother making other changes to the fork? I hate it when forks diverge substantially from the original version, as that makes merging upstream more tedious.